lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ @use '../../scss-shared/nx-header-variables';
+
+.nx-toast-container {
+  grid-area: content;
+  position: static;
+  justify-self:end;
+  margin-right: 8px;
+  width: 500px;
+  z-index: 1060;
+  overflow: hidden;
+  pointer-events: none;
+}
+
+// in legacy layouts where page scrolling is enabled, position: fixed helps keep
+// NxToastContainer in place. Top refers to the set height of the the header
+.nx-html--page-scrolling {
+  .nx-toast-container {
+    position: fixed;
+    top: nx-header-variables.$main-header-height;
+  }
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+  right: 0;
+  opacity: 1;
+  overflow: hidden;
+  margin-top: var(--nx-spacing-2x);
+
+  &.nx-toast--closing {
+    animation-name: animationOut;
+  }
+
+  .nx-alert {
+    margin: 0;
+    pointer-events:auto;
+  }
+}
+
+// The 74px in each animation refers to the total height of each toast: the content itself,
+// plus the padding and margin-top applied. Setting the max-height of each toast to 74px within
+// the animation allows for each subsequent toast to smoothly move up or down as toasts are
+// added and removed.
+@keyframes animationIn {
+  0% {
+    max-height: 0px;
```

Comment:
You mentioned having an undesirable bounce when you animated the margin.  Could we look into that again?  With the margin not animated, I'm pretty sure the spacing above the previous top toast instantly becomes 16px when a new toast is added, before gradually growing to make room for the new toast

Reply:
> You mentioned having an undesirable bounce when you animated the margin. Could we look into that again? With the margin not animated, I'm pretty sure the spacing above the previous top toast instantly becomes 16px when a new toast is added, before gradually growing to make room for the new toast

So I took a look at this again, and oddly it appears not to be the case. Even upon slowing the animation right down to 10% speed, the container still naturally expands. I did test adding the margins back in, and removing the 8px of margin from the max-height variable, but it seems as though to make it function correctly, I need to define the margin-top inside the animationIn at 40%, and inside .nx-toast. Since the animation appears to be smooth already, I've left it as is.

Reply:
> So I took a look at this again, and oddly it appears not to be the case. Even upon slowing the animation right down to 10% speed, the container still naturally expands

Observe the difference between 
![image](https://user-images.githubusercontent.com/3630091/187441059-32b36a9f-a7f8-4425-908e-751c8fc0fdbb.png)
where just a single toast is open, and 
![image](https://user-images.githubusercontent.com/3630091/187441193-6651a709-c52c-48cb-9da1-a954ea3eb3f0.png)

Which is the first frame of the animation of a second toast opening.  The margin of the second toast has been added instantaneously, doubling the amount of space above the first toast.  Don't get me wrong, this is basically imperceptible at full speed, but it's not what we intended so we might as well fix it.  

> to make it function correctly, I need to define the margin-top inside the animationIn at 40%, and inside .nx-toast

Right.  Seems straightforward to fix IMO.